views/components/ChartList.jsx

```diff
@@ -45,54 +48,78 @@ const ChartList = () => {
-            <TableCell align='center'>
-              <Button
-                type='button'
-                variant='text'
-                onClick={() => onDuplicateChart(_id)}
-              >
-                <PlaylistAdd />
-              </Button>
-            </TableCell>
-            <TableCell align='center'>
-              <Link href={routes.editChart(_id)} color='textPrimary'>
-                <Edit />
-              </Link>
-            </TableCell>
-            <TableCell align='center'>
-              <Button
-                type='button'
-                variant='text'
-                onClick={() => removeChart(_id)}
-              >
-                <Delete />
-              </Button>
-            </TableCell>
+    <>
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableCell align='center'>Title</TableCell>
+            <TableCell align='center'>Description</TableCell>
+            <TableCell align='center'>Duplicate</TableCell>
+            <TableCell align='center'>Edit</TableCell>
+            <TableCell align='center'>Share</TableCell>
+            <TableCell align='center'>Delete</TableCell>
           </TableRow>
-        ))}
-      </TableBody>
-    </Table>
+        </TableHead>
+        <TableBody>
+          {chartList.map((chart) => (
+            <TableRow key={chart._id}>
+              <TableCell align='center'>
+                <Link color='textPrimary' href={routes.chart(chart._id)}>
+                  {chart?.title?.toUpperCase()}
+                </Link>
+              </TableCell>
+              <TableCell align='center'>
+                {chart?.description?.toUpperCase()}
+              </TableCell>
+              <TableCell align='center'>
+                <Button
+                  type='button'
+                  variant='text'
+                  onClick={() => onDuplicateChart(chart._id)}
+                >
+                  <PlaylistAdd />
+                </Button>
+              </TableCell>
+              <TableCell align='center'>
+                <Link href={routes.editChart(chart._id)} color='textPrimary'>
+                  <Edit />
+                </Link>
+              </TableCell>
+              <TableCell align='center'>
+                <Button
+                  type='button'
+                  variant='text'
+                  onClick={() => handleShareChart(chart)}
+                >
+                  <Share />
+                </Button>
+              </TableCell>
+              <TableCell align='center'>
+                <Button
+                  type='button'
+                  variant='text'
+                  onClick={() => removeChart(chart._id)}
+                >
+                  <Delete />
+                </Button>
+              </TableCell>
+            </TableRow>
+          ))}
+        </TableBody>
+      </Table>
+      <ShareChart
+        open={!!chartState._id}
+        owner={user.uid}
```

Comment:
are we sure the user is the owner?

Reply:
yes, the user uid is set in the session and we pass it as the user object.
https://github.com/AMP-SCZ/dpdash/blob/8d829f10697a554b78945b95f3f56672127684a7/server/utils/userFromRequestUtil.js#L1-L11

Reply:
right, but couldn't the owner of the chart be a different user? Like if I'm viewing a chart that was shared with me, I am not the owner.

Reply:
yes, the charts object has an owner property, which could be named differently to maybe ownedBy? 
This commit updates this portion. https://github.com/AMP-SCZ/dpdash/pull/131/commits/4b7a005d45471fc07236b428f61a2f58b4284d6a